config/daemonconfig/daemonconfig.go

```diff
@@ -33,6 +33,7 @@ type DaemonConfig interface {
 	FillAuth(kc *auth.PassKeyChain)
 	StorageBackend() (StorageBackendType, *BackendConfig)
 	UpdateMirrors(mirrorsConfigDir, registryHost string) error
+	GetImageHost() string
```

Comment:
Seems there is no caller for the added method `GetImageHost`. Please remove it if so

Reply:
Done.